pkg/sdk/views_def.go

```diff
@@ -58,23 +59,33 @@ var viewDetails = g.PlainStruct("ViewDetails").
 	OptionalBool("Check").
 	OptionalText("Expression").
 	OptionalText("Comment").
-	OptionalText("PolicyName")
+	OptionalText("PolicyName").
+	OptionalText("PrivacyDomain")
 
 var viewColumn = g.NewQueryStruct("ViewColumn").
-	Text("Name", g.KeywordOptions().DoubleQuotes().Required()).
-	OptionalTextAssignment("COMMENT", g.ParameterOptions().SingleQuotes().NoEquals())
+	Text("Name", g.KeywordOptions().Required().DoubleQuotes()).
+	OptionalQueryStructField("ProjectionPolicy", viewColumnProjectionPolicy, g.KeywordOptions()).
+	OptionalQueryStructField("MaskingPolicy", viewColumnMaskingPolicy, g.KeywordOptions()).
+	OptionalTextAssignment("COMMENT", g.ParameterOptions().SingleQuotes().NoEquals()).
+	OptionalTags()
 
 var viewColumnMaskingPolicy = g.NewQueryStruct("ViewColumnMaskingPolicy").
-	Text("Name", g.KeywordOptions().Required()).
 	Identifier("MaskingPolicy", g.KindOfT[SchemaObjectIdentifier](), g.IdentifierOptions().SQL("MASKING POLICY").Required()).
-	NamedListWithParens("USING", g.KindOfT[string](), nil). // TODO: double quotes here?
-	OptionalTags()
+	NamedListWithParens("USING", g.KindOfT[string](), nil) // TODO: double quotes here?
+
+var viewColumnProjectionPolicy = g.NewQueryStruct("ViewColumnProjectionPolicy").
+	Identifier("ProjectionPolicy", g.KindOfT[SchemaObjectIdentifier](), g.IdentifierOptions().SQL("PROJECTION POLICY").Required())
 
 var viewRowAccessPolicy = g.NewQueryStruct("ViewRowAccessPolicy").
 	Identifier("RowAccessPolicy", g.KindOfT[SchemaObjectIdentifier](), g.IdentifierOptions().SQL("ROW ACCESS POLICY").Required()).
 	NamedListWithParens("ON", g.KindOfT[string](), g.KeywordOptions().Required()). // TODO: double quotes here?
 	WithValidation(g.ValidIdentifier, "RowAccessPolicy")
 
+var viewAggregationPolicy = g.NewQueryStruct("ViewAggregationPolicy").
+	Identifier("AggregationPolicy", g.KindOfT[SchemaObjectIdentifier](), g.IdentifierOptions().SQL("AGGREGATION POLICY").Required()).
+	NamedListWithParens("ENTITY KEY", g.KindOfT[string](), g.KeywordOptions()). // TODO: double quotes here?
+	WithValidation(g.ValidIdentifier, "AggregationPolicy")
```

Comment:
Hmm all identifiers other than `name` should have `g.ValidIdentifierIfSet` validation, otherwise we won't be able to tell difference between one error and the other (same in other validations I saw you add here).

Reply:
Fixed manually after discussion about limitations in the sdk.